resteasy-core/src/main/java/org/jboss/resteasy/core/ResteasyProviderFactoryImpl.java

```diff
@@ -1160,44 +1158,8 @@ private void addStringParameterUnmarshaller(Class<? extends StringParameterUnmar
       if (resolvers == null)
          return null;
       List<ContextResolver> rtn = new ArrayList<ContextResolver>();
-
       List<SortedKey<ContextResolver>> list = resolvers.getPossible(type);
-      if (type.isWildcardType())
-      {
-         // do it upside down if it is a wildcard type:  Note: this is to pass the stupid TCK which prefers that
-         // a wildcard type match up with other wildcard types
-         //         for (int i = list.size() - 1; i >= 0; i--)
-         //         {
-         //            rtn.add(list.get(i).obj);
-         //         }
-
-         // Fix for RESTEASY-1609.
-         // This is related to the fix in RESTEASY-1471, prior to which user ContextResolvers appeared
-         // to be built-in. The original loop may have been in response to that bug, so the reversal
-         // may not be necessary. In any case, this code will do the reversal but put user ContextResolvers
-         // at the beginning of the list.
-         for (int i = list.size() - 1; i >= 0; i--)
-         {
-            if (!list.get(i).isBuiltin)
-            {
-               rtn.add(list.get(i).obj);
-            }
-         }
-         for (int i = list.size() - 1; i >= 0; i--)
-         {
-            if (list.get(i).isBuiltin)
-            {
-               rtn.add(list.get(i).obj);
-            }
-         }
-      }
-      else
-      {
-         for (SortedKey<ContextResolver> resolver : list)
-         {
-            rtn.add(resolver.obj);
-         }
-      }

```

Comment:
Hi @NicoNes.

I'm trying to understand why you took this section out. I don't see any broken tests, but could you explain your thinking?

Thanks,
Ron